Assets/Scripts/Game/UserInterfaceWindows/UIWindowFactory.cs

```diff
@@ -42,7 +42,8 @@ public enum UIWindowType
         QuestOffer,
         Rest,
         SpellBook,
-        SpellMaker,
+        SpellIconPicker = WitchesCovenPopup + 1, // new enum: setting this to the value after WitchesCovenPopup to avoid breaking precompiled mods
+        SpellMaker = SpellBook + 1,
```

Comment:
Please could you change this to:
`
SpellIconPicker = 42,
SpellMaker = 24,
`

Thanks.